### PR TITLE
Picking XLSX Report

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: stock_picking_report_xlsx/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Known issues / Roadmap
+======================
+
+ * No known issues.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/stock-logistics-reporting/issues>`_.
```
Comment (tafaRU) [inline stock_picking_report_xlsx/README.rst]: Please use up-to-date version.

File: stock_picking_report_xlsx/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/stock-logistics-reporting/issues/new?body=module:%20stock_picking_partner_lang%0Aversion:%208.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+- Serpent Consulting Services Pvt. Ltd.<jay.vora@serpentcs.com>
```
Comment (tafaRU) [inline stock_picking_report_xlsx/README.rst]: replace - by *

File: stock_picking_report_xlsx/README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (tafaRU) [inline stock_picking_report_xlsx/README.rst]: s/http/https

File: stock_picking_report_xlsx/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```
Comment (tafaRU) [inline stock_picking_report_xlsx/__init__.py]: replace © by Copyright

File: stock_picking_report_xlsx/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```
Comment (tafaRU) [inline stock_picking_report_xlsx/__openerp__.py]: Same as above

File: stock_picking_report_xlsx/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock picking xlsx report',
+    'version': '8.0.0.1.0',
```
Comment (tafaRU) [inline stock_picking_report_xlsx/__openerp__.py]: Use 8.0.1.0.0 instead.

File: stock_picking_report_xlsx/report/__init__.py [hunk 7]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```
Comment (tafaRU) [inline stock_picking_report_xlsx/report/__init__.py]: Same as above.

File: stock_picking_report_xlsx/report/__init__.py [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import stock_picking_xlsx_report
```
Comment (tafaRU) [inline stock_picking_report_xlsx/report/__init__.py]: no newline at end of file

File: stock_picking_report_xlsx/report/stock_picking_xlsx_report.py [hunk 9]
```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```
Comment (tafaRU) [inline stock_picking_report_xlsx/report/stock_picking_xlsx_report.py]: Same as above.

File: stock_picking_report_xlsx/static/.svn/all-wcprops [hunk 10]
```diff
@@ -0,0 +1,5 @@
+K 25
```
Comment (tafaRU) [inline stock_picking_report_xlsx/static/.svn/all-wcprops]: Please remove this file.

File: stock_picking_report_xlsx/static/.svn/entries [hunk 11]
```diff
@@ -0,0 +1,31 @@
+10
```
Comment (tafaRU) [inline stock_picking_report_xlsx/static/.svn/entries]: Please remove this file.

File: stock_picking_report_xlsx/views/report_view.xml [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+             id="stock_piccking_xlsx"
+             model="stock.picking"
+             string="Print to XLSX"
+             report_type="xlsx"
+             name="stock.picking.xlsx"
+             file="stock.picking.xlsx"
+             attachment_use="False"
+         />
+    </data>
+</openerp>
```
Comment (tafaRU) [inline stock_picking_report_xlsx/views/report_view.xml]: no newline at end of file

**Review by tafaRU — CHANGES_REQUESTED** [review]
In addition, please replace Serpent icon by https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.png

**Comment by tafaRU** [comment]
@JayVora-SerpentCS, thanks a lot!
Could you also please squash commits to only one?

**Comment by JayVora-SerpentCS** [comment]
Commits Squashed,
Thanks.

File: stock_picking_report_xlsx/__openerp__.py [hunk 13]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock picking xlsx report',
+    'version': '8.0.1.0.0',
+    'summary': "Print stock picking report in xlsx",
+    'website': 'http://www.serpentcs.com',
+    'author': """Serpent Consulting Services Pvt. Ltd.,
```
Comment (tafaRU) [inline stock_picking_report_xlsx/__openerp__.py]: better do not use triple quote here and please also add Agile Business Group.

File: stock_picking_report_xlsx/__openerp__.py [hunk 14]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock picking xlsx report',
+    'version': '8.0.1.0.0',
+    'summary': "Print stock picking report in xlsx",
```
Comment (tafaRU) [inline stock_picking_report_xlsx/__openerp__.py]: for consintency it should be better use single quote everywhere, so please put:
`'Print stock picking report in xlsx'`

**Review by tafaRU — APPROVED** [review]
Thanks a lot!

File: stock_picking_report_xlsx/README.rst [hunk 15]
```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=============================
+Stock Picking Rerport in XLSX
```
Comment (eLBati) [inline stock_picking_report_xlsx/README.rst]: Rerport -> Report

**Review by eLBati — CHANGES_REQUESTED** [review]

File: stock_picking_report_xlsx/README.rst (hunk 15 above)
Comment (eLBati) [reply stock_picking_report_xlsx/README.rst]: Afterwards, :+1:

**Comment by JayVora-SerpentCS** [comment]
Would someone check please?

**Comment by rousseldenis** [comment]
@JayVora-SerpentCS At least we can merge. Can you just squash the commits ?

**Comment by JayVora-SerpentCS** [comment]
Did it, finally! so old PR gets a life!

**Comment by rousseldenis** [comment]
@eLBati Could you update your review ?

**Comment by eLBati** [comment]
@JayVora-SerpentCS @rousseldenis too many commits and files changed :thinking:

**Comment by rousseldenis** [comment]
> @JayVora-SerpentCS @rousseldenis too many commits and files changed

Indeed.

@JayVora-SerpentCS Could you fix this and limit changed files to module you want ?

**Comment by JayVora-SerpentCS** [comment]
I just did upstream as it was too old! I preserved all commits.

**Pull request closed without merging**